web/package.json

```diff
@@ -105,6 +105,9 @@
     "url": "https://github.com/electricitymap/electricitymap-contrib.git"
   },
   "lint-staged": {
+    "*": [
+      "prettier --write ."
+    ],
```

Comment:
Good point!
I'll check if that syntax work (it should).

Reply:
The syntax did work locally (with a slight change) but failed in lint-staged because it explicitly pass files to prettier. But changing the lint-staged matching to the following seems to work as expected:
```json
"!(*.{js,jsx,ts,tsx})": [
      "prettier --write"
    ],
```